test/cases/aws.sh

```diff
@@ -199,9 +199,14 @@ greenprint "Pulling cloud-image-val container"
 if [[ "$CI_PROJECT_NAME" =~ "cloud-image-val" ]]; then
   # If running on CIV, get dev container
   TAG=${CI_COMMIT_REF_SLUG}
+  if [ -z "${CIV_OPTIONS}" ]; then
```

Comment:
This will fail with `CIV_OPTIONS: unbound variable` since this script has `set -e`. I suggest to update it to this:
```suggestion
  if [ -z "${CIV_OPTIONS:-}" ]; then
```

Reply:
But it should fail if `CIV_OPTIONS` is not defined. Some of them are not optional